versions-maven-plugin/src/it/it-dependency-updates-aggregate-report-issue-333/invoker.properties

```diff
@@ -1 +1 @@
-invoker.goals=site
+invoker.goals = ${project.groupId}:${project.artifactId}:${project.version}:dependency-updates-aggregate-report
```

Comment:
Does it will work with site plugin?
Or only as direct report call?

Reply:
Yes, it does. The only difference is the location of the generated files (site vs reports).